frontend/src/Pages/SignUp.jsx

```diff
@@ -1,122 +1,101 @@
-
-import React, { useState } from 'react'
-import { useNavigate } from 'react-router-dom'
-import "./SignUp.css"
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import "./SignUp.css";
 
 const SignUp = () => {
-
-const [name,setName]=useState("")
-const [email,setEmail]=useState("")
-const [password,setPassword]=useState("")
-// const [pass2,setPass2]=useState("")
-
-
-
-
-const handleSubmit=()=>{
-  const payload={
-    name,email,password
-  }
-  //console.log(payload)
-  //we are connectiong FE to BE
-  fetch("http://localhost:8800/users/register",{
-    method:"POST",
-    headers:{
-      "Content-Type":"application/json"
-    },
-    body:JSON.stringify(payload)
-  }).then(res=>res.json())
-  .then(res=>console.log(res))
-  .catch(err=>console.log(err))
-  
-  
-  setName("")
-  setEmail("")
-  setPassword("")
- 
-
-}
-
-
-
-
-
-const navigate=useNavigate()
-const Loginpage=()=>{
-    navigate("/login")
-
-
-
-
-}
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  // const [pass2,setPass2]=useState("")
+
+  const handleSubmit = () => {
+    const payload = {
+      name,
+      email,
+      password,
+    };
+    //console.log(payload)
+    //we are connectiong FE to BE
+    fetch("http://localhost:8800/users/register", {
+      method: "POST",
```

Comment:
sigunp will not work. YOu have hard-coded the localhost URL. It will not work when you deploy the changes.